monarchmoney/monarchmoney.py

```diff
@@ -401,14 +407,31 @@ async def get_transactions(
                 "categories": category_ids,
                 "accounts": account_ids,
                 "tags": tag_ids,
-                "hasAttachments": has_attachments,
-                "hasNotes": has_notes,
-                "hideFromReports": hidden_from_reports,
-                "isRecurring": is_recurring,
-                "isSplit": is_split,
             },
         }
 
+        # If bool filters are not defined (i.e. None), then it should not apply the filter
+        if has_attachments is not None:
+            variables["filters"]["hasAttachments"] = has_attachments
+
+        if has_notes is not None:
+            variables["filters"]["hasNotes"] = has_notes
+
+        if hidden_from_reports is not None:
+            variables["filters"]["hideFromReports"] = hidden_from_reports
+
+        if is_recurring is not None:
+            variables["filters"]["isRecurring"] = is_recurring
+
+        if is_split is not None:
+            variables["filters"]["isSplit"] = is_split
+
+        if imported_from_mint is not None:
+            variables["filters"]["importedFromMint"] = is_split
```

Comment:
Looks like this should be 'imported_from_mint'

Reply:
Ah thanks for catching that! Just fixed it.